FAIL
run: wait(exec("multi-recurse")) returned 0: FAILED
